practice/src/App.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+  Navigate,
+} from 'react-router-dom';
+import SignInForm from './components/DataDisplay/Authen/SignIn/SignIn';
+import SignUpForm from './components/DataDisplay/Authen/SignUp/SignUp';
```

Comment:
@ngocdiepphan Can we re-structure and use `import SignInForm from './components/DataDisplay/Authen/SignIn';`?